doc/installation_instructions/basic.rst

```diff
@@ -0,0 +1,300 @@
+******************
+Basic Installation
+******************
+
+The following will show how to install both SmartSim and SmartRedis.
+
+.. note::
+  For users on platforms with a 'site install' of SmartSim please follow
+  :ref:`these instructions <site install>`.
+
+=============
+Prerequisites
+=============
+
+The base prerequisites to install SmartSim and SmartRedis are:
+
+  - Python 3.7-3.9
+  - Pip
+  - Cmake 3.13.x (or later)
+  - C compiler
+  - C++ compiler
+  - GNU Make > 4.0
+  - git
+  - git-lfs
```

Comment:
On https://www.craylabs.org/docs/installation.html you include how to install git-lfs but do not add it here, but the format of the pages are similar, should we add a link to git-lfs installation or make a note

Reply:
Made this a link to the install instructions